README.md

```
@@ -35,10 +35,32 @@ Agent tools for testing documentation procedures and validating that documented
+
+   ```text
+   /doc-detective:init
+   ```
+
+### Codex, Cursor, OpenCode, and other AI coding assistants
+
+#### Option 1: Install with `npx skills`
 
 > [!WARNING]
-> `npx skills` only installs skills, not agents, commands, or other tools. For full functionality, consider [manual installation](#option-3-manual-installation).
+> `npx skills` only installs skills, not agents, commands, or other tools. For full functionality, consider [manual installation](#option-4-manual-installation).
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)